phpBB/styles/prosilver/theme/stylesheet.css

```diff
@@ -7,6 +7,7 @@
     --------------------------------------------------------------
 */
 
+@import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css");
```

Comment:
agree, it will not even cache this way. We need a way to utilize the cdn when we can but also as you stated include it when no connection is present. I just did not know how the assets worked